spec/features/providers/allocations/edit_initial_allocation_spec.rb

```diff
@@ -19,8 +19,10 @@
       and_i_click_change
 
       then_i_see_do_you_want_page
+      and_i_click_continue
+      then_i_see_an_error_message
 
-      when_i_select_yes
+      when_i_select_yes_with_error
```

Comment:
Is this renaming is necessary? I'm not clear how adding `_with_error` helps 🤔

Reply:
The id for the button element actually changes when the error message gets added to it - this means the prism page object no longer works for that button. I did try adding another element to the page object, but because they both use the same "Yes" value, it was getting all kinds of issues...